CMake/SMTKConfig.cmake.in

```diff
@@ -0,0 +1,21 @@
+#Setup the CMake module paths
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "@SMTK_MODULE_DIR@")
+
+#When building documentation internally we re-import ourselves, which
+#doesnt work and generates CMake warnings. So we will only look for SMTK
+#if the smtkCore target doesn't already exist
+if(TARGET smtkCore)
+else()
+
+  set(SMTK_USE_SYSTEM_MOAB @SMTK_USE_SYSTEM_MOAB@)
+  if(SMTK_USE_SYSTEM_MOAB)
+    find_package(MOAB REQUIRED)
```

Comment:
Shouldn't we also be saving `MOAB_ROOT_DIR` to the installed `SMTKConfig.cmake` so that find_package(MOAB) always finds the **same** MOAB that SMTK was built with?